moped-editor/src/views/projects/projectView/SubprojectFundingModal.js

```diff
@@ -135,7 +135,7 @@ const SubprojectFundingModal = ({
             },
           }}
           options={{
-            ...(data.length < 11 && {
+            ...(filteredData.length < 11 && {
```

Comment:
the table rendered filteredData, updating pagination to also refer to the same array